psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -718,7 +716,7 @@ def scf_print_energies(self):
     #else:
     #    self.set_variable(self.functional().name() + ' TOTAL ENERGY', dft_energy)  # overwritten later for DH
 
-    self.set_variable('SCF ITERATIONS', self.iteration_)
+    self.set_variable("SCF ITERATIONS", self.iteration_)  # P::e SCF
```

Comment:
There are a _lot_ of changes like this. Can you explain why this is necessary, and when we should add comments like this to a `set_variable` line?

Reply:
we need to harvest qcvars from py-side and need to know what module they're assoc. with, so this comment is the marker for both. I've added some notes in the "docs 19" commit. good catch -- doesn't look like any of the scraping conventions were documented.